packages/shared-ux/button_toolbar/src/toolbar/toolbar.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { ReactElement } from 'react';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+
+import { IconButtonGroup, PrimaryButton } from '../buttons';
+import { ToolbarPopover } from '../popover';
+
+interface NamedSlots {
```

Comment:
This ends up getting exported as part of the Props interface, so it probably needs docs and inline docs per parameter:

```
  /** Some description **/
  primaryButton: ...
```